ts/features/itwallet/common/components/ItwSkeumorphicCard/CardClaim.tsx

```diff
@@ -72,30 +76,43 @@ const CardClaim = ({
         E.fold(constNull, decoded => {
           if (SimpleDateClaim.is(decoded)) {
             const formattedDate = decoded.toString(dateFormat);
-            return <ClaimLabel {...labelProps}>{formattedDate}</ClaimLabel>;
+            return (
+              <ClaimLabel {...labelProps} valuesHidden={valuesHidden}>
```

Comment:
`valuesHidden` could be omitted, it is passed by the `labelProps` object
```suggestion
              <ClaimLabel {...labelProps}>
```